physical_validation/integrator.py

```diff
@@ -56,6 +56,7 @@ def convergence(
         test is implemented:
         `max_deviation`
     verbose : bool, optional
+        If True, some output is printed
```

Comment:
Maybe use "If True, print more detailed output"

(There is always some output).

Reply:
Yeah that's better. I was just stealing the docstring from pymbar but certainly that was not informative.